pkg/scheduler/framework/plugins/nodevolumelimits/csi_test.go

```diff
@@ -746,7 +746,7 @@ func getFakeCSINodeLister(csiNode *storagev1.CSINode) fakeframework.CSINodeListe
 	return csiNodeLister
 }
 
-func getNodeWithPodAndVolumeLimits(limitSource string, pods []*v1.Pod, limit int64, driverNames ...string) (*framework.NodeInfo, *storagev1.CSINode) {
+func getNodeWithPodAndVolumeLimits(limitSource string, pods []*v1.Pod, limit int32, driverNames ...string) (*framework.NodeInfo, *storagev1.CSINode) {
```

Comment:
Please revert this, let's focus on the deprecation only. I think it's on purpose for int64 as a limit value.

Reply:
Fair enough, it is a separate concern. I noticed while handling the `ptr` migration that the value ends up stored in an `int32` field, so this runs the risk of silent truncation (albeit in test code where all values are much, much smaller than the `int32` limits).

Reply:
Yes, that can be a separate PR if you like.

Reply:
Done as #120690.